src/backend/bs_publish

```diff
@@ -1505,6 +1505,74 @@ sub clean_blobdir {
   rmdir($blobdir);
 }
 
+sub createrepo {
+  my ($projid, $repoid, $data) = @_;
+  my $prp = "$projid/$repoid";
+  print "    creating repository\n";
+  my ($extrep, $stageservers) = BSUrlmapper::get_extrep_stageservers($prp);
+  return unless $extrep;
```

Comment:
you don't  create repo metadata anymore, if no stage server is defined?
This is not wanted....

Reply:
You suggest to use `BSUrlmapper::get_extrep($prp)` here, as done in the updated patch ?